integration_tests/src/main/python/data_gen.py

```diff
@@ -541,6 +541,35 @@ def make_tuple():
     def contains_ts(self):
         return any(child[1].contains_ts() for child in self.children)
 
+class StructGen2(DataGen):
```

Comment:
Why do we need/want this?  You can do the same thing with a function.

```
def my_struct_gen(child1, child2, nullable=True, special_cases=[]):
  return StructGen([('a', child1),('b', child2),('c', child2)], nullable=nullable, special_cases=special_cases)
```

Reply:
removed